apps/alarm/app.js

```diff
@@ -69,25 +78,27 @@ function showMainMenu() {
 function editDOW(dow, onchange) {
   const menu = {
     '': { 'title': /*LANG*/'Days of Week' },
-    /*LANG*/'< Back' : () => onchange(dow)
+    /*LANG*/'< Back': () => onchange(dow)
   };
-  for (let i = 0; i < 7; i++) (i => {
-    let dayOfWeek = require("locale").dow({ getDay: () => i });
-    menu[dayOfWeek] = {
-      value: !!(dow&(1<<i)),
+  var dows = require("date_utils").dows(firstDayOfWeek);
+  for (var i = 0; i < 7; i++) (i => {
```

Comment:
I'm merging anyway, but IMO `require("date_utils").dows(firstDayOfWeek).forEach(...)` is a bit tidier than `for (var i = 0; i < 7; i++) (i => { ... })(i)` now we have an array to work from